source/jormungandr/jormungandr/__init__.py

```diff
@@ -68,7 +68,7 @@
 
 from jormungandr.instance_manager import InstanceManager
 
-i_manager = InstanceManager(ini_files=app.config.get('INI_FILE', None),
+i_manager = InstanceManager(conf_files=app.config.get('INI_FILE', None),
```

Comment:
I was not sure to know every use of that parameter.

I know the tests use it, but I'll need to ask @kinnou02 if it's the only use.